fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -20,6 +20,8 @@
       flush_interval 5s
       max_retry_wait 300
       disable_retry_limit true
+      buffer_type "#{ENV['BUFFER_TYPE'] || 'memory'}"
+      buffer_path "#{ENV['FILE_BUFFER_PATH_ES_COPY'] || '/dev/null'}"
```

Comment:
If a user sets `BUFFER_TYPE` and then fails to set `FILE_BUFFER_PATH_ES_COPY`, and `FILE_BUFFER_PATH_ESOPS_COPY`, and all the others, won't they still default to `/dev/null`?  The `BUFFER_TYPE` parameter affects all uses of a fluentd `buffer`, so we might want to handle that case.